presto-hive/src/main/java/com/facebook/presto/hive/HiveUtil.java

```diff
@@ -238,6 +262,13 @@ static String getInputFormatName(Properties schema)
         return name;
     }
 
+    public static String getSerializationClassName(Properties schema)
```

Comment:
Let me clarify the changes in this class. The change in line 195 must be placed in this class since it calls ``getRecordReader`` directly and we don't have the chance to rename the property, once we use ``HiveMultiInputFormat`` directly and remove ``com.facebook.presto.twitter.hive.thrift.ThriftGeneralInputFormat``  which is just a mirror of EB's ``HiveMultiInputFormat``. The change on ``HiveMultiInputFormat`` has been merged in EB so we will remove line 249 to line 252 once the EB make a new release. The method ``getSerializationClassName`` could be moved to someplace out of this class. Then rest changes are included in PR #7916 so they will go to OSS.